xdsl/interpreters/riscv.py

```diff
@@ -518,6 +518,17 @@ def run_fld(
         results = ((args[0] + offset).float64[0],)
         return RiscvFunctions.set_reg_values(interpreter, op.results, results)
 
+    @impl(riscv.FMvDOp)
+    def run_fmv_d(
+        self,
+        interpreter: Interpreter,
+        op: riscv.FMvDOp,
+        args: tuple[Any, ...],
+    ):
+        args = RiscvFunctions.get_reg_values(interpreter, op.operands, args)
+        results = args
```

Comment:
I think you could get away without this line?

Reply:
Defo, I'm just trying to make it explicit what's happening here, I feel like it makes it a bit clearer that it's on purpose. Maybe it could be resolved better with a comment...